src/update_complete.js

```diff
@@ -0,0 +1,29 @@
+/* eslint-disable import/extensions */
+import { getData, updateIndexes, updateTasks } from './services';
+
+export function updateCompleteTask(id, isCompleted = false) {
+  if (id === null) return;
+
+  if (typeof id === 'string') id = Number(id);
+
+  let tasks = getData('tasks');
+
+  tasks = tasks.map((task) => {
+    if (id === task.index) {
+      return { ...task, completed: isCompleted };
+    }
+
+    return task;
+  });
+
+  updateTasks(tasks);
+}
```

Comment:
- [ ] Kindly add unit test for this function as it is part of this milestone's requirements.
![image](https://user-images.githubusercontent.com/44057863/179314354-32f31486-4621-4e6d-8062-7af924672bcd.png)